circle.yml

```diff
@@ -41,24 +41,25 @@ test:
     # See the circleCI documentation regarding parallelism
     # to understand how multiple containers can be used to
     # run subsets of tests in parallel.
-    - ./scripts/all-tests.sh:
-        timeout: 900  # if a command runs this many seconds without output, kill it
-        parallel: true
+    - echo All tests are disabled now.  # TODO: Enable tests when we're ready.
```

Comment:
Not necessarily true. All sort of magic syntax is applied here with YAML/Bash, adding quotes might just break it! It's weird, I know!

The [latest build](https://circleci.com/gh/Edraak/edx-platform/83) contained something like this:
```
 - echo "TODO: Enable all tests."
```

But [it broke because of the colon](https://circleci.com/gh/Edraak/edx-platform/83)!